src/main/resources/config.sk

```diff
@@ -338,6 +338,11 @@ databases:
 
 		backup interval: 2 hours
 
+		backup purge: false #set to "true" to enable the automatic purging feature
+		purge required: 50 #Number of backup files required in order to start the purge process
+		purge remain: 20 #Number of files to be left remaining when purge starts
```

Comment:
imo it's easier to explain if the config option is how many files to purge instead of the number of files are left over after purging